build/tools/avalanchejs/README.md

```diff
@@ -5,29 +5,34 @@ AvalancheJS is a JavaScript Library for interfacing with the [Avalanche](../../.
 The APIs currently supported by default are:
 
 * Admin API
+* Auth API
 * AVM API \(X-Chain\)
+* EVM API \(C-Chain)
```

Comment:
Should this be `\(C-Chain\)` ?
Fwiw on my markdown editor it renders OK even without the `\`